csharp/AppEncryption/AppEncryption.Tests/AppEncryption/Persistence/DynamoDbMetastoreImplTest.cs

```diff
@@ -276,6 +283,7 @@ private void TestPrimaryBuilderPath()
             // Hack to inject default region since we don't explicitly require one be specified as we do in KMS impl
             AWSConfigs.AWSRegion = "us-west-2";
             DynamoDbMetastoreImpl dbMetastoreImpl = NewBuilder(Region)
+                .WithEndPointConfiguration("http://localhost:" + DynamoDbPort, Region)
```

Comment:
This test is now functionally equivalent to the test that follows. Do we need both?

Reply:
reverted this test back to the original commented out version. Think it would be good to leave it in there so we can uncomment and verify that the primary builder path is not broken when we make any new changes to the Dynamo implementation.